# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sn9c102/linux-2.6.18.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur sn9c102-1.26-orig/sn9c102_core.c sn9c102-1.26/sn9c102_core.c
--- sn9c102-1.26-orig/sn9c102_core.c	2006-01-11 03:12:54.000000000 +0200
+++ sn9c102-1.26/sn9c102_core.c	2006-09-21 00:05:41.000000000 +0300
@@ -2580,11 +2580,11 @@
 	case VIDIOC_G_CTRL:
 		return sn9c102_vidioc_g_ctrl(cam, arg);
 
-	case VIDIOC_S_CTRL_OLD:
+//	case VIDIOC_S_CTRL_OLD:
 	case VIDIOC_S_CTRL:
 		return sn9c102_vidioc_s_ctrl(cam, arg);
 
-	case VIDIOC_CROPCAP_OLD:
+//	case VIDIOC_CROPCAP_OLD:
 	case VIDIOC_CROPCAP:
 		return sn9c102_vidioc_cropcap(cam, arg);
 
@@ -2631,7 +2631,7 @@
 	case VIDIOC_G_PARM:
 		return sn9c102_vidioc_g_parm(cam, arg);
 
-	case VIDIOC_S_PARM_OLD:
+//	case VIDIOC_S_PARM_OLD:
 	case VIDIOC_S_PARM:
 		return sn9c102_vidioc_s_parm(cam, arg);
 
